src/components/UnstyledLink/UnstyledLink.tsx

```diff
@@ -1,20 +1,9 @@
 import React from 'react';
 import {unstyled} from '../shared';
-import {usePolaris} from '../../hooks';
+import usePolaris from '../../utilities/use-polaris';
+import {Props as UnstyledLinkProps} from '../../utilities/unstyled-link';
 
-export interface Props extends React.HTMLProps<HTMLAnchorElement> {
-  /** A destination to link to */
-  url: string;
-  /** Forces url to open in a new tab */
-  external?: boolean;
-  /** Tells the browser to download the url instead of opening it. Provides a hint for the downloaded filename if it is a string value. */
-  download?: string | boolean;
-  /**	Content to display inside the link */
-  children?: React.ReactNode;
-  [key: string]: any;
-}
-
-export type LinkLikeComponent = React.ComponentType<Props> | undefined;
+interface Props extends UnstyledLinkProps {}
```

Comment:
Needed for styleguide parsing

Reply:
Should we create an issue in polaris-styleguide and link to that? This is something we'll want to fix eventually and an issue can gather context over time as it progresses